packages/api/tests/es/test-stats.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
```

Comment:
This is by no means exhaustive, but there are currently no tests for `es/stats` and I started the debugging by starting to write this test file. I plan to continue writing tests, but would like to get the fix merged in.